### ISSUE TITLE
do something

### ISSUE THREAD

**Issue by edenvidal** [rnbwdev/rnbw#797]
![Image](https://github.com/rnbwdev/rnbw/assets/13418616/13878db2-316b-4708-91a5-645f0cfd0333)

move "download" and "save" to jumpstart under "projects"
in jumpstart, change "projects" into "project"
remove "turn into" as it is becoming a custom action
remove "design"

**Comment by edenvidal** [comment]
@maxdnc, check this out.

**Comment by edenvidal** [comment]
@atulbhatt-system32 this is easy. let's close this (:

**Comment by atulbhatt-system32** [comment]
@edenvidal how to access turn into after removing it and same thing for design. Do we need to move them to some other place?

**Comment by edenvidal** [comment]
So, as we discussed, all custom actions will be inside "Others". right? so there's a tailored group for custom actions. if there's a file = actions shows up in the group 🙌

**Comment by edenvidal** [comment]
@atulbhatt-system32 so, as we discussed, all custom actions will be inside "Others". right? so there's a tailored group for custom actions. if there's a file = actions shows up in the group 🙌

**Comment by edenvidal** [comment]
yes. we don't need it

**Comment by edenvidal** [comment]
@atulbhatt-system32 yes it shows in the do something

we can keep "code" and "design" since they are toggling the panels

**Comment by atulbhatt-system32** [comment]
@edenvidal for now I have moved Turn Into to other group in do something menu.

However, I tried to list all the actions from actions folder.
I was able to display all the actions, but as these are custom hooks they need to be instantiated before using and they have to instantiate at the initial render of the page.

This connecting actions with UI is kinda extension thing which we haven't discussed yet. How will it hook to the UI. Also how it will hook to the shortcuts on stage. We only defined building actions. Currently we use them similar to API. We use the action hook wherever its is required in the UI.

For ex:
Turn Into opens a dialog box to pick in what tag to turn into and after that the actual action gets executed.
